### DISCUSSION TITLE
playing some clips does not work

### DISCUSSION THREAD

**Discussion by wrohdewald** [blakeblackshear/frigate#9087] [Ask A Question]
Hi,

this is about the web GUI.

Most clips always play fine. But about one out of 10-20 does not. There are only minutes between working and non-working  records, and I did not change anything in between.

If I go to the camera event list and click on the picture, playing does not start. It just hangs forever.

For the same recording, all of this works:

- http://myserver:5000/recording/ne/2023-12-27/13/46/49
- Homeassistant Media
- kodi (NFS).

I enabled debugging output but nothing relevant shows up. Restarting frigate does not help. The recording sits on an ordinary local hard disk drive.

So - any ideas how I could debug this? Before fixing the cause, I would like to know I can get relevant logging messages.

```
logger:
  default: debug
```

**Comment by twister36** [comment]
Do you get any error or warning in your main frigate log?

**Comment by wrohdewald** [comment]
Here it comes. But there is no error message.

```
Dec 28 11:54:37 speicher docker-compose[1874570]: #033[36mfrigate    |#033[0m 2023-12-28 11:54:37.504876417  10.210.137.29 - - [28/Dec/2023:11:54:37 +0100] "GET /vod/event/1703681214.319963-u4e3fj/master.m3u8 HTTP/1.1" 200 155 "http://10.210.137.3:5000/events?cameras=ne" "Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/115.0" "-"
Dec 28 11:54:37 speicher docker-compose[1874570]: #033[36mfrigate    |#033[0m 2023-12-28 11:54:37.513915569  10.210.137.29 - - [28/Dec/2023:11:54:37 +0100] "GET /vod/event/1703681214.319963-u4e3fj/index-v1.m3u8 HTTP/1.1" 200 191 "http://10.210.137.3:5000/events?cameras=ne" "Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/115.0" "-"
Dec 28 11:54:37 speicher docker-compose[1874570]: #033[36mfrigate    |#033[0m 2023-12-28 11:54:37.702784226  10.210.137.29 - - [28/Dec/2023:11:54:37 +0100] "GET /vod/event/1703681214.319963-u4e3fj/seg-1-v1.ts HTTP/1.1" 200 23008568 "http://10.210.137.3:5000/events?cameras=ne" "Mozilla/5.0 (X11; Linux x86_64; rv:109.0) Gecko/20100101 Firefox/115.0" "-"
Dec 28 11:54:38 speicher docker-compose[1874570]: #033[36mfrigate    |#033[0m 2023-12-28 11:54:38.728767903  [2023-12-28 11:54:38] asyncio                        DEBUG   : Using selector: EpollSelector
```
